.claude/commands/tm/workflows/auto-implement-tasks.md

```
@@ -0,0 +1,107 @@
+- Performance considerations
+
+### 4. **Progressive Implementation**
+
+Step-by-step with validation:
+
+```
+Step 1/5: Setting up component structure ✓
+Step 2/5: Implementing core logic ✓
+Step 3/5: Adding error handling ⚡ (in progress)
+Step 4/5: Writing tests ⏳
+Step 5/5: Integration testing ⏳
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)